app/src/test/java/org/simple/clinic/scanid/ScanSimpleIdEffectHandlerTest.kt

```diff
@@ -241,4 +241,26 @@ class ScanSimpleIdEffectHandlerTest {
     testCase.assertOutgoingEvents(OnlinePatientLookupWithIdentifierCompleted(results, identifier))
     verifyNoMoreInteractions(uiActions)
   }
+
+  @Test
+  fun `when save complete medical records effect is received, then save the patient medical records received`() {
```

Comment:
Can you rename the test to.. `when save complete medical records effect is received, then save the patient medical records`